iroha/src/sumeragi.rs

```diff
@@ -10,17 +17,17 @@ use std::{
 };
 
 trait Consensus {
-    fn round(&mut self, transactions: Vec<Transaction>) -> Option<Block>;
+    fn round(&mut self, transactions: Vec<Transaction>) -> Option<PendingBlock>;
 }
 
 #[derive(Io, Decode, Encode, Debug, Clone)]
 pub enum Message {
     /// Is sent by leader to all validating peers, when a new block is created.
-    Created(Block),
+    Created(SignedBlock),
     /// Is sent by validating peers to proxy tail and observing peers when they have signed this block.
-    Signed(Block),
+    Signed(SignedBlock),
```

Comment:
The naming seems to be a bit controversial here. The block is in the process of being signed - it is not signed by all members yet.

Reply:
It means it's signed at least once - when all signs are in place, should we name it "Voted" or smthg?

Reply:
Ok, I think we can leave it as it.